src/client/components/SubmitFormSingleInputComponent/SubmitFormSingleInput.js

```diff
@@ -16,7 +21,20 @@ export default function SingleInputFormComponent({
         type="textbox"
         placeholder={value}
       />
-      <ButtonComponent ButtonTitle={ButtonTitle} />
+      <ButtonComponent
+        ButtonTitle={ButtonTitle}
+        onClick={() => {
+          const inputvalue = document.getElementsByName('.form-input').value;
```

Comment:
Also not correct "react way". Please add a todo.

Reply:
```js
...
  const [modalShown, toggleModalShown] = useState('modalbox');

  function toggleModal() {
    toggleModalShown((modalshown) => {
      return modalShown === 'modalbox' ? 'modalbox showmodal' : 'modalbox';
    });
  }

  return (
    <div className="modalcontainer">
      <SubmitButton onClick={toggleModal} ButtonTitle="Open modal" />
      <ListModalPopup
        toggleModal={modalShown}
 ...
```

In modal component:
```js
export default function ListModalPopup(props) {
  const {
    toggleModal,
    modalTitle,
    ButtonTitle,
    optionA,
    optionB,
    radiobox,
    onchangeradiobox,
    radiobox2,
    onchangeradiobox2,
  } = props;

  console.log(toggleModal);
  return (
    <div className={toggleModal} id="modalbox">
...
```

Reply:
@priyankapandey13 
The way you do it is shown above. Could you make those changes. Also remember not to rename components on import
There are currently lots of PopUp components referring to various different functions. Export and import should exactly the same

Reply:
Done, Thanks.